tiiuae/ghaf-givc
# Port more of givc admin functionality

Now it should be on-par with golang implementation

How to run it in debug/test mode?
`cargo run --bin givc-admin -- --addr 127.0.0.1 --port 9000`
to run admin

`cargo run --bin givc-agent -- --name bogus --addr 127.0.0.1 --port 9001 --type 1 --subtype 14 --admin-server-addr 127.0.0.1 --admin-server-port 9000`
(to run agent -- you can see it registering in admin,  agent is self-reg stub only, all methods raise unimplemented)
